### PR TITLE
Release v1.6.0

### PR DESCRIPTION
**Pull request by shopiwags**
- Remove the old alipay icon (`alipay`). (https://github.com/activemerchant/payment_icons/pull/408)

- Rename the new alipay icon from `alipay_cn` to `alipay` to replace the old version. (https://github.com/activemerchant/payment_icons/pull/408)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -72,9 +72,9 @@ GEM
     coderay (1.1.3)
     concurrent-ruby (1.1.8)
     crass (1.0.6)
-    dedup (0.1.2)
+    dedup (0.1.3)
```
Comment (shopiwags) [inline Gemfile.lock:75]: My `bundle install` modified `dedup` and `ffi`, not sure if we should keep or remove.

Comment (larouxn) [reply Gemfile.lock:75]: Let's remove both random gem changes.

**Review by larouxn — CHANGES_REQUESTED** [review]
Let's release this as `v1.6.0` as it's semi-breaking.

File: Gemfile.lock [hunk 2]
```diff
@@ -72,9 +72,9 @@ GEM
     coderay (1.1.3)
     concurrent-ruby (1.1.8)
     crass (1.0.6)
-    dedup (0.1.3)
+    dedup (0.1.2)
```
Comment (larouxn) [inline Gemfile.lock]: Let's drop this change.

File: Gemfile.lock [hunk 3]
```diff
@@ -72,9 +72,9 @@ GEM
...
     erubi (1.10.0)
-    ffi (1.14.2)
+    ffi (1.13.1)
```
Comment (larouxn) [inline Gemfile.lock]: Let's drop this change.

**Review by larouxn — COMMENTED** [review]
Pretty much good to go. Let's aim for a Monday/Tuesday release next week.

**Review by larouxn — APPROVED** [review]

**Pull request merged**